content/posts/welcome.md

```diff
@@ -1,13 +1,12 @@
 +++
-title = "Welcome to the updated Jellyfin.Org"
+title = "Welcome to the New Website!"
```

Comment:
it feels wrong to edit blog posts posted a while ago...

Reply:
I’m okay with it, since the URL doesn’t change.

Reply:
I don't plan on making large changes to posts, just minor things like clean titles, or spelling improvements. Changing posts isn't inherently bad though, because the entire history will always be available in this repository.

Reply:
That requires people who want to make sure nothing was changed to browse here and test, which is suboptimal. But whatever... :)